cli.py

```diff
@@ -1936,29 +1936,29 @@ def returnGroupInfo(self, userlist):
             1 = we've errored, exit with error string
             2 = we've got work yet to do, onto the next stage
         """
-        for strng in userlist:
+        for string in userlist:
```

Comment:
"string" is the name of a module from python's standard library. Naming a variable "string" is not a good idea, as it will shadow the module name. I believe this "typo" is intentional.